build.sbt

```diff
@@ -50,12 +50,15 @@ lazy val cli = project
 lazy val spark = project
   .in(file("spark"))
   .settings(commonSettings)
-  .settings(
-    libraryDependencies ++= Seq(
-      "commons-io" % "commons-io" % "2.6",
-      "com.typesafe.scala-logging" %% "scala-logging" % "3.9.0",
-      "log4j" % "log4j" % "1.2.17"
-    ) ++ sparkDependencies)
+  .settings(libraryDependencies ++= Seq(
+    "commons-io" % "commons-io" % "2.6",
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.9.0",
+    "log4j" % "log4j" % "1.2.17",
+    "com.amazonaws" % "aws-java-sdk" % "1.11.542" % Test,
+    "org.apache.hadoop" % "hadoop-common" % "2.9.2" % Test,
+    "org.apache.hadoop" % "hadoop-aws" % "2.9.2" % Test,
+    "io.findify" %% "s3mock" % "0.2.5" % Test
```

Comment:
We can remove the `s3mock` dependency now. Also, I don't think we need `aws-java-sdk` or `hadoop-aws` any more either?